### PR TITLE
Update `.travis.yml` for automated deployment

### LINKED ISSUE #78: Automatic deployment

### PR DESCRIPTION
**Pull request by ydlamba**
**Update `.travis.yml` for automated deployment**

@domoritz You need to put your Github Token here. Then we need to test this.

Fixes #78.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
 cache: yarn
+deploy:
+  provider: pages
+  skip-cleanup: true
+  github-token: $GITHUB_TOKEN  # Set in the settings page of your repository, as a secure variable (Over to you, dom)
+  keep-history: true
+  local-dir: dist
+  name: Dominik Moritz
+  email: domoritz@gmail.com
+  project-name: vega-editor
```
Comment (domoritz) [inline .travis.yml]: Isn't the name vega/editor or just editor?

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,24 @@
...
 node_js:
   - "8"
-
 script:
   - yarn run lint
-
+  - yarn run generate-example-images
+  - yarn run build
 cache: yarn
+deploy:
```
Comment (domoritz) [inline .travis.yml]: Do we need to say that we only want to do this on master and only for tagged commits?

File: .travis.yml (hunk 1 above)
Comment (ydlamba) [reply .travis.yml]: Let's make it by the repo name i.e. `editor`. (Although not necessary)

File: .travis.yml (hunk 2 above)
Comment (ydlamba) [reply .travis.yml]: Tagged commits makes more sense. Updating it.

File: .travis.yml [hunk 3]
```diff
@@ -1,8 +1,24 @@
...
+  local-dir: dist
+  name: Dominik Moritz
+  email: domoritz@gmail.com
+  project-name: vega-editor
+  addons:
+    apt:
+      packages:
+        - parallel
+  on:
+    branch: master
```
Comment (domoritz) [inline .travis.yml]: add `tags: true`

File: .travis.yml (hunk 2 above)
Comment (domoritz) [reply .travis.yml]: see below

File: .travis.yml (hunk 1 above)
Comment (domoritz) [reply .travis.yml]: Can you just delete it? It's not really necessary.

**Pull request merged**